external-crates/move/crates/move-analyzer/prettier-plugin/tests/run_spec.js

```diff
@@ -24,10 +24,13 @@ global.run_spec = function(dirname) {
                 parser: 'move-parse',
             })
             return result.then((formatted_prog) => {
-                if (!fs.existsSync(exp_path) || UB == 1) {
-                    // expected file does not exist or user asked it to be regenerated - create and
-                    // populate it
+                if (UB == 1) {
+                    // user asked to regenerate output
                     fs.writeFileSync(exp_path, formatted_prog, 'utf8');
+                    return;
+                }
+                if (!fs.existsSync(exp_path)) {
+                    assert.fail('\n' + formatted_prog + "No expected output file")
```

Comment:
```suggestion
                    assert.fail(`\n${formatted_prog}No expected output file`)
```
Don't think it's worth changing in this PR, maybe just worth doing going forward--I've found it more common to use template literals in TS for this sort of thing.

Reply:
Changed these in this file as well so that I don't forget how to do it in the future :-)